src/misc/cache/lv_image_header_cache.c

```diff
@@ -87,18 +83,17 @@ void lv_image_header_cache_drop(const void * src)
     };
 
     lv_cache_drop(img_header_cache_p, &search_key, NULL);
-#else
-    LV_UNUSED(src);
-#endif
+}
+
+bool lv_image_header_cache_is_disabled(void)
```

Comment:
Avoid negative logic/return values, use instead positive values => clean code suggestion.
Having negative return values requires extra mental effort if used like this:
`if (lv_image_header_cache_is_disabled() == false) {...}`

```suggestion
bool lv_image_header_cache_is_enabled(void)
```